specification/common-types/resource-management/v5/types.json

```diff
@@ -624,6 +624,74 @@
           "type": "string"
         }
       }
+    },
+    "UsageName": {
+      "type": "object",
+      "description": "Name of the usage metrics",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "The name of the usage metric."
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "The localized name of the usage metric."
+        }
+      }
+    },
+    "Usage": {
+      "type": "object",
+      "description": "Resource usage and limit.",
+      "properties": {
+        "unit": {
```

Comment:
missing "id" as an optional property.  You don't have to return it but its part of the official contract and needs to be in the schema.  https://eng.ms/docs/products/arm/api_contracts/usagesapicontract